docker-compose.yml

```diff
@@ -33,3 +33,13 @@ services:
       - ${DEVELOPMENT_DIR}/repositories/graphext/node_modules/@blueprintjs:/root/graphext/blueprintjs
     ports:
       - "9999:9999"
+  popover:
+    image: node:12
+    command: sh -c 'cd packages/popover2 && yarn compile:css && cp -R lib/css ../../../root/graphext/blueprintjs/popover2/lib'
+    working_dir: /hostdata
+    volumes:
+      - .:/hostdata
+      - ${DEVELOPMENT_DIR}/config/npmrc:/root/.npmrc
+      - ${DEVELOPMENT_DIR}/repositories/graphext/node_modules/@blueprintjs:/root/graphext/blueprintjs
+    ports:
+      - "9999:9999"
```

Comment:
Why do we need this command? Maybe we can use some documentation in the graphext readme about how we can use it!

Reply:
This command is to make the dev easier between Blueprint and Graphext/Storybook. 
I have added some comments about this in `GRAPHEXT_README.md` and Storybook readme too.